bundle/nxiq-operator-certified.clusterserviceversion.yaml

```diff
@@ -122,15 +151,15 @@ spec:
     | `iq.adminPort`       | Port of the application connector. Must match the value in the `configYaml` property | `8071`            |
     | `iq.memory`          | The amount of RAM to allocate                                | `1Gi`             |
     | `iq.licenseSecret`   | The base-64 encoded license file to be installed at startup  | `""`              |
-    | `iq.configYaml`      | A YAML block which will be used as a configuration block for IQ Server. | See `values.yaml` |
+    | `iq.configYaml`      | A YAML block which will be used as a configuration block for IQ Server. | See example YAML when creating an instance. |
```

Comment:
where is this example YAML? Is it printed on the console or something when they start up?